kafka/src/main/java/com/metamx/tranquility/kafka/util/PropertiesHelper.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.metamx.tranquility.kafka.util;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Throwables;
+import com.google.common.collect.Sets;
+import com.metamx.common.IAE;
+import com.metamx.common.logger.Logger;
+import com.metamx.tranquility.kafka.model.TopicSchemaPair;
+import org.apache.commons.lang.Validate;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+/**
+ * Read and validate application settings from a properties file.
+ */
+public class PropertiesHelper
```

Comment:
I think it'd be good to use the same tranquility properties that `tranquility-server` does (https://github.com/druid-io/tranquility/blob/master/docs/server.md#configuration-reference). That stuff should probably be pulled out into core as it looks like more than one module will use it.

By "that stuff" I mean:

- `DataSourceConfig` in com.metamx.tranquility.server.config
- The code in com.metamx.tranquility.server.http.Main.createServlet that creates tranquilizers from a set of `DataSourceConfig`s